assets/src/styles/blocks/core-overrides/Image.scss

```diff
@@ -1,12 +1,10 @@
 @mixin rounded-image-size($size) {
-  figure,
+  border-radius: 50%;
```

Comment:
This change causes this [issue](https://www-dev.greenpeace.org/test-oberon/51128-2/)

Reply:
I've also added the `.force-no-caption` class to the selector, since the issues you noticed occur when there is a caption. This class is now present in all our patterns, so it should work where we expect it!